sycl/include/sycl/builtins.hpp

```diff
@@ -1377,6 +1377,198 @@ detail::common_rel_ret_t<T> signbit(T x) __NOEXC {
       __sycl_std::__invoke_SignBitSet<detail::internal_rel_ret_t<T>>(x));
 }
 
+// marray relational functions
+template <typename T,
+          typename = detail::enable_if_t<
+              detail::is_mgenfloat<T>::value &&
+                  detail::is_svgenfloat<detail::marray_element_type<T>>::value,
+              T>>
```

Comment:
Since you are not using this type anyway, you can omit the second argument to use default `void`